pkg/config/config_local.go

```diff
@@ -0,0 +1,77 @@
+// +build !remoteclient
```

Comment:
This means Podman has to use this too but libpod is using `ABISupport`. @baude, cool with that?

Reply:
yeah i think it should probably be ABI support ... or be fenced by os !linux

Reply:
`!linux` would be my favourite option.

Reply:
decided to use `remote`